Configs/.config/hypr/scripts/gpuinfo.sh

```diff
@@ -1,7 +1,55 @@
 #!/bin/bash
 
-# Get the GPU temperature using nvidia-smi
-gpu_info=$(nvidia-smi --query-gpu=temperature.gpu,utilization.gpu,clocks.current.graphics,clocks.max.graphics,power.draw,power.limit --format=csv,noheader,nounits)
+# Check for NVIDIA GPU using nvidia-smi
+nvidia_gpu=$(nvidia-smi --query-gpu=gpu_name --format=csv,noheader,nounits | head -n 1)
+
+# Find all GPUs using lspci
+all_gpus=$(lspci -nn | grep -E 'VGA|3D controller' | grep -oE '\[....:....\]' | tr -d '[]')
+
+# Initialize variables for integrated GPUs
+intel_gpu=""
+amd_gpu=""
+
+# Loop through all GPUs to identify integrated ones
+for gpu in $all_gpus; do
+  gpu_info=$(lspci -v -s "$gpu")
```

Comment:
the script works fine but its giving this error for this `lspci -v -s "$gpu"` line -

![image](https://github.com/prasanthrangan/hyprdots/assets/106020512/29ad199f-02a2-4623-b044-b27db6d84f51)

Reply:
I'm fixing this now -- Its just a generic error as the bus hasn't been specified by that point in the script.

Reply:
I apologize for the inconvenience. The script can still identify GPUs without specifying the bus number. Here's the updated script -- Should be updated and ready to merge

Reply:
hey no worries, thanks for the contribution as always!